File: onebot/src/main/java/cn/evole/onebot/sdk/event/notice/group/GroupLuckyKingNoticeEvent.java
package cn.evole.onebot.sdk.event.notice.group;

import cn.evole.onebot.sdk.event.notice.NoticeEvent;
import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class GroupLuckyKingNoticeEvent extends NoticeEvent {

    @SerializedName( "sub_type")
    public String subType;

    @SerializedName( "group_id")
    public long groupId;

    @SerializedName( "target_id")
    public long targetId;


}

File: onebot/src/main/kotlin/sdk/event/notice/group/GroupLuckyKingNoticeEvent.kt
package cn.evolvefield.onebot.sdk.event.notice.group

import cn.evolvefield.onebot.sdk.event.notice.NoticeEvent
import com.google.gson.annotations.SerializedName

class GroupLuckyKingNoticeEvent : NoticeEvent() {
    @SerializedName("sub_type")
    var subType = ""
    @SerializedName("group_id")
    var groupId = 0L
    @SerializedName("target_id")
    var targetId = 0L
}
